grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_form.ts

```diff
@@ -24,9 +26,25 @@ export class OsqueryForm extends FlowArgumentForm<OsqueryArgs> implements
     query: new FormControl(this.defaultQueryDisplayed, Validators.required),
     timeoutMillis: new FormControl(null, Validators.required),
     ignoreStderrErrors: new FormControl(null),
+    fileCollectColumns: new FormControl([]),
```

Comment:
Minor wording correction: `fileCollectionColumns` would sounds a bit more natural. If you prefer to use a verb, I'd go with: `collectFileCollumns`.